Dockerfile

```diff
@@ -27,9 +27,7 @@ RUN curl -OL https://github.com/gohugoio/hugo/releases/download/v${VERSION_HUGO}
     rm -rf hugo_${VERSION_HUGO}_Linux-64bit.tar.gz
 
 # Configure environment
-RUN echo export PATH="\
-    /root/.nvm/versions/node/${VERSION_NODE}/bin:\
-    $PATH" >> ~/.bashrc && \
+RUN echo export PATH="/root/.nvm/versions/node/${VERSION_NODE}/bin:\$PATH" >> ~/.bashrc && \
```

Comment:
Hi! Thank you for your PR. :)

I think the backslash \ will lead to unexpected results since it is escaping the $ character in $PATH.